docs/get-started/terminology.md

```diff
@@ -10,3 +10,6 @@ This document provides clear definitions of key concepts used in **Concrete** fr
 
 * **Circuit:** The result of compilation. A circuit includes both client and server components. It has methods for various operations, such as printing and evaluation.
 
+* **Table lookup (TLU):** Table loopup (TLU) stands for instructions of the form `y = T[i]`. In practice, in FHE, they are done with programmable bootstrapping, which is the equivalent operation on encrypted values.
```

Comment:
```suggestion
* **Table Lookup (TLU):** TLU stands for instructions in the form of `y = T[i]`. In the practice of FHE, this operation is performed with Programmable Bootstrapping, which is the equivalent operation on encrypted values.
```

Reply:
"In the practice of FHE"? it looks not to be what I meant and it looks strange

Reply:
@yuxizama tell me if we can change this, please

Reply:
Do you mean "In Concrete, this operation is performed with Programmable Bootstrapping" ?
I just find "**In** practice, **in** FHE..." is a little bit repetitive

Reply:
Yes I agree it's repetitive but "In the practice of FHE" looks to have a different meaning to me. Let me propose something

Reply:
"* **Table Lookup (TLU):** TLU stands for instructions in the form of `y = T[i]`. In FHE, this operation is performed with Programmable Bootstrapping, which is the equivalent operation on encrypted values."

Good for you @yuxizama ?

Reply:
Yes perfect, thank you!